R/reporter-progress.R

```diff
@@ -57,6 +58,16 @@ ProgressReporter <- R6::R6Class("ProgressReporter",
       self$show_header()
     },
 
+    start_file = function(file) {
+      self$file_name <- file
+    },
+
+    start_test = function(context, test) {
+      if (is.null(context)) {
+        context(self$file_name)
```

Comment:
Using the regular `context()` function rather than `self$start_context()` or `self$.start_context()` because for the multi-reporter we need to assign `self$.context` in the multi-reporter rather than in the contained reporters.

Reply:
this one as well